kvstore/query/get_query.py

```diff
@@ -0,0 +1,30 @@
+from kvstore.query.base_query import BaseQuery
+from kvstore.utils.enums import QueryStatus, QueryType
+
+
+class GetQuery(BaseQuery):
+    """Get query"""
+
+    def __init__(self, key):
+        super().__init__()
+        self.key = key
+        self.status = QueryStatus.QUERY_CREATED
+        self.query_type = QueryType.GET
+        self.dbts = []
+
+    def execute(self):
+        """Executes a get query and returns the corresponding value"""
+
+        # Look up which read replica to read from
+        raise NotImplementedError("Not yet implemented")
+
+    def _get_nearest_read_replica(self, key):
+        """
+        Given a key, return the best read replica location
+        """
```

Comment:
this is going to be problematic, missing a notImplementedError, or a `pass` statement